src/components/userauth/registerHelper.js

```diff
@@ -0,0 +1,12 @@
+export function validateEmail(email) {
+  // regex source https://emailregex.com/
+  const re = /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+  return re.test(email);
+}
+
+export const createErrorClassName = (error) => {
+  if (!error) {
+    return null;
```

Comment:
you most likely want to return undefined here, otherwise if you do `className={null}` it does `className="null"` which is annoying.

Reply:
className=undefined causes it to be empty in snapshots. you can google this too

Reply:
PTAL